src/pages/index.js

```diff
@@ -9,11 +9,13 @@ import "../global.scss"
 const App = (props) => {
   return (
     <main className = 'body'>
-      <h1>Football Timer</h1>
-      <GameClock/>
-      <Team name = "Team One" id ={1}/>
-      <Team name = "Team Two" id = {2}/>
-      <ThrowTimer/>
+      <container style={{width: 350}}>
```

Comment:
There is no such thing as a container tag to my knowledge it's best to not create your own html tags, but instead use what is available like divs, or create a component called Container

Reply:
Lol somehow I assumed that containers that I heard people talking about were made with a container tag lol. Back to divs it is